views/standard/spec/indicators.js

```diff
@@ -296,6 +308,15 @@ export default {
    }
 };
 
+function getAgeColor(days) {
+   switch (Math.floor(days / 3)) {
+      case 0: return 'green';
+      case 1: return 'yellow';
```

Comment:
```suggestion
      case 1: return '#fabd02';
```

`yellow` is difficult to see on the white background. This color I suggested seems better and it looks fine in dark mode, too.